common/test/acceptance/pages/studio/import_export.py

```diff
@@ -0,0 +1,198 @@
+"""
+Course Export page.
+"""
+from bok_choy.promise import EmptyPromise
+import os
+import requests
+from .utils import click_css
+from .library import LibraryPage
+from .course_page import CoursePage
+from . import BASE_URL
+
+
+class ExportMixin(object):
+    """
+    Export page Mixin.
+    """
+
+    url_path = "export"
+
+    def is_browser_on_page(self):
+        """
+        Verify this is the export page
+        """
+        return self.q(css='body.view-export').present
+
+    def _get_tarball(self, url):
+        """
+        Download tarball at `url`
+        """
+        kwargs = dict()
+        session_id = [{i['name']: i['value']} for i in self.browser.get_cookies() if i['name'] == u'sessionid']
+        if session_id:
+            kwargs.update({
+                'cookies': session_id[0]
+            })
+
+        response = requests.get(url, **kwargs)
+
+        return response.status_code == 200, response.headers
+
+    def download_course(self):
```

Comment:
Can you name this method something else since it works for both course and library exports?